code/stable/hghc/SRS/mdBook/Makefile

```diff
@@ -0,0 +1,7 @@
+build:
+	mdbook build
+
+server:
+	mdbook serve --open
+
+.PHONY: build server
```

Comment:
Some of our other generated Makefiles have a watermark on them containing some Drasil information. e.g.,

https://github.com/JacquesCarette/Drasil/blob/e8797118de04997a450458cbfd6e98f881559275/code/stable/dblpend/src/java/Makefile#L1-L5

Do you think we can/should add them here too?

Reply:
Note: we don't necessarily need to add Project Name/Purpose, but the Drasil-generated watermark is nice, I think.

Reply:
Added in https://github.com/JacquesCarette/Drasil/pull/3848/commits/798bdf23e25ccb25cb954454f67ca8868d58e588. Also added the watermark for the TeX Makefiles.